subsystems/intakesubsystem.py

```diff
@@ -0,0 +1,122 @@
+from enum import Enum, auto
+from commands2 import Subsystem
+
+from util.simtalon import Talon
+from util.simneo import NEOBrushless
+from util.simcoder import CTREEncoder
+import constants
+
+
+class IntakeSubsystem(Subsystem):
+    class IntakeState(Enum):
+        Intaking = (
+            auto()
+        )  # arm is down, motor is running inwards/holding depending on note
+        Holding = auto()  # arm in feeding position, motor holding
+        Feeding = auto()  # arm in feeding position, motor ejecting
+        Staging = auto()  # arm is in position to score amp/trap, motor holding
+        Amp = auto()  # arm in same staging position, motor ejects
+        Trap = auto()  # arm pushes into trap, delay(?), motor ejects
+
+    def __init__(self) -> None:
+        Subsystem.__init__(self)
+        self.setName(__class__.__name__)  # basic subsystem boilerplate
+
+        self.pivotEncoder = CTREEncoder(
+            constants.kPivotEncoderID,
+            constants.kIntakeAngleOffset.degrees(),
+        )
+
+        self.intakeMotor = NEOBrushless(
+            constants.kIntakeCANID,
+            constants.kIntakeName,
+            constants.kIntakePIDSlot,
+            constants.kIntakePGain,
+            constants.kIntakeIGain,
+            constants.kIntakeDGain,
+            constants.kIntakeInverted,
+        )
+        self.pivotMotor = Talon(
+            constants.kPivotCANID,
+            constants.kPivotName,
+            constants.kPivotPGain,
+            constants.kPivotIGain,
+            constants.kPivotDGain,
+            constants.kPivotInverted,
+        )
+
+        # pivot motor spins 60 times per arm revolution
+        # use absolute encoder to determine motor position
+        pivotMotorPosition = (
+            self.pivotEncoder.getPosition().degrees()
+            / constants.kDegeersPerRevolution
+            * 60
+        )
+        self.pivotMotor.setEncoderPosition(pivotMotorPosition)
+
+        self.state = self.IntakeState.Holding
+
+    def periodic(self) -> None:
+        # get actual velocity values for intake motor later
+
+        if self.state == self.IntakeState.Intaking:
+            self.pivotMotor.set(
+                Talon.ControlMode.Position,
+                constants.kFloorPositionAngle.degrees()
+                / constants.kDegeersPerRevolution,
+            )
+            if self.intakeMotor.getLimitSwitch(NEOBrushless.LimitSwitch.Forwards):
+                self.intakeMotor.setNeutralOutput(NEOBrushless.NeutralMode.Brake)
```

Comment:
bad docs, but setNeutralOutput sets what the mode for neutral output should be
you are looking for motor.neutralOutput()